client/jetpack-connect/user-type/form.jsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class UserTypeForm extends Component {
+	static propTypes = {
+		submitForm: PropTypes.func.isRequired,
+
+		// from localize() HoC
+		translate: PropTypes.func.isRequired,
+	};
+
+	state = { userType: '' };
+
+	handleRadioChange = event => this.setState( { userType: event.currentTarget.value } );
+
+	handleSubmit = userType => {
+		this.props.recordTracksEvent( 'calypso_signup_actions_submit_user_type', {
+			value: userType,
+		} );
+
+		this.props.submitForm( userType );
+	};
+
+	getUserTypeQuestions = () => {
+		const { translate } = this.props;
+
+		return [
+			{
+				id: 1,
```

Comment:
Why don't we just use the `slug`s? The only purpose of `id`s seems to be for displaying them, and the slugs will be unique enough for that purpose.